spec/models/redirection_spec.rb

```diff
@@ -21,6 +21,17 @@
     end
   end
 
+  describe ".in_ring_order" do
+    it "returns redirections in the order they appear next in the ring" do
+      create(:redirection)
+      first = Redirection.order(id: :asc).first
```

Comment:
Excellent point, this line is fine then.

> On Dec 8, 2015, at 7:05 PM, Edward Loveall <notifications@github.com> wrote:
> 
> In spec/models/redirection_spec.rb:
> 
> > @@ -21,6 +21,17 @@
> >      end
> >    end
> >  
> > +  describe ".in_ring_order" do
> > +    it "returns redirections in the order they appear next in the ring" do
> > +      create(:redirection)
> > +      first = Redirection.order(id: :asc).first
> True, but I think it shows some intent here. I want the record with the lowest numbered ID rather than "whatever the first one by some metric" is.
> 
> —
> Reply to this email directly or view it on GitHub.
>